### PR TITLE
Add file

### PR DESCRIPTION
**Pull request by s34rching**

### CODE REVIEW

File: client/to-show-file.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const asd = '4'
```
Comment (s34rching) [inline client/to-show-file.js:1]: BULLSHIT

File: client/to-show-file.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+const qwerty = '5'
```
Comment (s34rching) [inline client/to-show-file.js:2]: Oh man, so nice!

File: client/to-show-file.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+const zxc = '6'
```
Comment (s34rching) [inline client/to-show-file.js:3]: Do we really need this?

**Review by s34rching — COMMENTED** [review]
Just a comment

File: client/to-show-file.js (hunk 1 above)
Comment (Glushkow) [inline client/to-show-file.js:1]: test

Comment (Glushkow) [reply client/to-show-file.js:1]: test2

**Review by Glushkow — APPROVED** [review]

**Pull request merged**